modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.js

```diff
@@ -14,6 +14,64 @@
 
 Liferay = window.Liferay || {};
 
+Liferay.Webpack = {
+	Container: {},
+
+	SharedScope: {},
+
+	require: function(moduleName, callback) {
+		var parts = moduleName.split("/");
+
+		var containerId = parts[0];
+		var path =
+			parts.length == 1 ?
```

Comment:
Always prefer `===` (because [we'll lint for it "soon"](https://github.com/liferay/liferay-frontend-projects/issues/14)).